src/Stratis.Sidechains.Networks/CirrusRegTest.cs

```diff
@@ -99,7 +100,8 @@ public CirrusRegTest()
             this.Federations = new Federations();
 
             // Default transaction-signing keys!
-            this.Federations.RegisterFederation(new Federation(federationPubKeys.ToArray()));
+            // Use the new keys as the old keys should never be used by the new opcode.
+            this.Federations.RegisterFederation(new Federation(newFederationPubKeys.ToArray()));
```

Comment:
Changed this back to the federation keys of the new federation as that is what the new opcode applies to.